lib/pop3.c

```diff
@@ -171,12 +172,15 @@ const struct Curl_handler Curl_handler_pop3s = {
 /* SASL parameters for the pop3 protocol */
 static const struct SASLproto saslpop3 = {
   "pop",                      /* The service name */
-  '*',                        /* Code received when continuation is expected */
-  '+',                        /* Code to receive upon authentication success */
-  255 - 8,                    /* Maximum initial response length (no max) */
   pop3_perform_auth,          /* Send authentication command */
   pop3_continue_auth,         /* Send authentication continuation */
-  pop3_get_message            /* Get SASL response message */
+  pop3_cancel_auth,           /* Send authentication cancellation */
+  pop3_get_message,           /* Get SASL response message */
+  255 - 8,                    /* Maximum initial response length (no max) */
```

Comment:
This comment is a bit strange. What does `(no max)` mean? Is this a copy-paste mistake? Maybe the comment should explain where 255 and 8 come from?

Reply:
Yes, cutn'paste too much!
Done, + a comment in curl_mime.h.